accounts/templates/account/login.html

```
@@ -0,0 +1,76 @@
+{% extends "core/layouts/box.html" %}
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)